test/setup/index.ts

```diff
@@ -90,7 +90,7 @@ async function predeploy(
     } catch (error) {
       console.log(`Namespace ${namespace} already exist`);
     }
-    const gcrDockercfg = process.env['PRIVATE_REGISTRIES_DOCKERCFG'] || '{}';
+    const gcrDockercfg = '{ "credsStore": "gcr" }';
```

Comment:
Would this `gcrDockercfg` handle pulling from private registry as well?

Reply:
Yesss, now the idea is not to pass username+password but have the credentials helper auto-generate them for us :)